README.md

```diff
@@ -375,31 +363,35 @@ Please execute at least step 1 + 2 if before importing restdocs-wiremock into yo
 
 Given that the `master` branch on the upstream repository is in the state from which you want to create a release, execute the following steps:
 
-```shell
-git checkout master
-git pull upstream master
+1. Create release via the GitHub UI
 
-# Print current version to the terminal
-./gradlew currentVersion
+https://github.com/ePages-de/restdocs-wiremock/releases/new
 
-# (a) Release new patch version
-./gradlew release -Prelease.versionIncrementer=incrementPatch
+Use the intended version number as "Tag version", e.g. "0.8.0".
 
-# (b) Release new minor version
-./gradlew release -Prelease.versionIncrementer=incrementMajor
+This will automatically trigger a Travis build which publishes the JAR files for this release to Sonatype.
 
-## (c) Release new major version
-./gradlew release -Prelease.versionIncrementer=incrementMajor
+2. Login to Sonatype
 
-# Print current version to the terminal
-./gradlew currentVersion
-```
+Login to Sonatype and navigate to the [staging repositories](https://oss.sonatype.org/#stagingRepositories).
+
+3. Close the staging repository
+
+Select the generated staging repository and close it.
+Check that there are no errors afterwards (e.g. missing signatures or Javadoc JARs).
+
+4. Release the repository
+
+Select the genereted staging repository and publish it.
+Soon after, the release should be available in the in the "Public Repositories" of ePages:
+
+https://oss.sonatype.org/service/local/repo_groups/public/content/com/epages/restdocs-wiremock/maven-metadata.xml
+
+(Steps 2 to 4 could be automated, if we want to. See [publish-plugin/README.md](https://github.com/gradle-nexus/publish-plugin/blob/master/README.md#publishing-to-maven-central-via-sonatype-ossrh))
```

Comment:
I think, it would be better to create a issue for this and not mention it here.

Reply:
I am not yet sure whether I think we should also automate the final steps or not. That's why I won't create an issue for this. My intention with this statement was to conserve the knowledge that it might easily be automated. If wanted, I can remove that, no problem.

Reply:
The issue can preserve the same knowledge, we don't have to _fix_ it. IMO, this unclutters the readme.

Reply:
I created https://github.com/ePages-de/restdocs-wiremock/issues/76 for this and linked it to this PR.